xml/deployment_remote.xml

```diff
@@ -96,7 +96,7 @@
   </para>
 
   <sect2 xml:id="sec-remote-installation-scenario-vncstat">
-   <title>Installation from DVD via VNC</title>
+   <title>Installation from Source Media via VNC</title>
```

Comment:
I can't tell if this should be singular or plural. If it could concievably use >1 volume then it can stay as "media". Otherwise, it should be "medium".

Reply:
yes, it can, let's leave it like that